packages/app/src/components/breadcrumb/index.tsx

```diff
@@ -8,17 +8,19 @@ export type Props = BaseProps & {
 };
 const Breadcrumb: React.FC<Props> = ({ on, list, className = '' }) => {
   return (
-    <div className={classnames(`text-thm-on-${on}`, className)}>
-      <div className="flex items-center gap-1">
-        {list.map((label) => (
-          <React.Fragment key={label}>
-            <div className="">{label}</div>
-            <div className="">
-              <ChevronRightIcon className="w-em" />
-            </div>
-          </React.Fragment>
-        ))}
-      </div>
+    <div
+      className={classnames(
+        `text-thm-${on} flex items-center gap-1`,
```

Comment:
I just fixed text color to text-thm-${on} from text-thm-on-${on}.
Other parts are refactoring, such as making the nesting shallower.